src/encoded/static/components/search.js

```diff
@@ -1063,14 +1063,16 @@ export class FacetList extends React.Component {
         return (
             <div className={`box facets${addClasses ? ` ${addClasses}` : ''}`}>
                 <div className={`orientation${this.props.orientation === 'horizontal' ? ' horizontal' : ''}`}>
-                    <div className="search-header-control">
-                        {context ? <DocTypeTitle searchResults={context} wrapper={children => <h1>{children} {docTypeTitleSuffix}</h1>} /> : null}
-                        {clearButton ?
-                            <div className="clear-filters-control">
-                                <a href={context.clear_filters}>Clear Filters <i className="icon icon-times-circle" /></a>
-                            </div>
-                        : null}
-                    </div>
+                    {(context && clearButton) ?
```

Comment:
Should this be `&&` or `||`? I’m not sure what the logic is supposed to be, but if it’s supposed to be `&&` then you can remove both conditionals inside the "search-header-control" div since you’ll know they’re both true.

Reply:
Oh yes, good catch. Should be ||. Thank you!